src/lib.rs

```diff
@@ -13,11 +13,12 @@
 //!
 //! Use macros from both crates:
 //!
-//! ```rust,ignore
-//! #[macro_use(lambda)]
+//! ```rust
+//! #[macro_use]
```

Comment:
this fixes a silly thing I noticed in the docs

![screen shot 2018-10-31 at 11 42 30 am](https://user-images.githubusercontent.com/2242/47800232-40b3d300-dd02-11e8-86ac-a6b9f12676f4.png)

rustdoc generates a likely too hash warning for ignored examples. This keeps the example as rendered with a hidden main so its runnable, removing the warning in rustdoc